### PR TITLE
Fixes version detection and add config file subscribe dependency

### PR DESCRIPTION
**Pull request by vrf888**

### CODE REVIEW

**Comment by juniorsysadmin** [comment]
@vrf888 Would prefer this as a 2 separate Pull Requests

File: manifests/service.pp [hunk 1]
```diff
@@ -15,9 +15,10 @@
 
   if $confluence::manage_service {
     service { 'confluence':
-      ensure  => 'running',
-      enable  => true,
-      require => [ Class['confluence::config'], File[$service_file_location], ],
+      ensure    => 'running',
+      enable    => true,
+      require   => Class['confluence::config'],
+      subscribe => File[$service_file_location],
```
Comment (juniorsysadmin) [inline manifests/service.pp]: Okay with this

File: lib/facter/confluence_version.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@
     if confluence_process.nil?
       'unknown'
     else
-      confluence_process.scan(%r{\d+\.\d+\.\d+}).first
+      confluence_process.scan(%r{atlassian-confluence-(\d+\.\d+\.\d+)}).first.first
```
Comment (juniorsysadmin) [inline lib/facter/confluence_version.rb]: Want verification on this. What confluence versions have you tested this fact with?

**Comment by oranenj** [comment]
For the version detection, I merged #124 instead, since it includes tests. @vrf888, could you do a rebase with only the subscribe change, please?

**Comment by bastelfreak** [comment]
Hi,
I'm going to close this PR due to inactivity. Please reopen it if you're still interested in it.

**Pull request closed without merging**